lib/periodicChecks.js

```diff
@@ -0,0 +1,43 @@
+const assignUnAssignedReviewers = async (pullRequest, context) => {
+  const assignees = pullRequest.assignees.map(user => user.login);
+  const reviewers = pullRequest.requested_reviewers.map(user => user.login);
+
+  let botComment = 'PTAL ';
+  const prParams = context.repo(
+    {
+      number: pullRequest.number,
+    }
+  );
+
+  const unAssignedReviewers = reviewers.filter(reviewer => {
+    if (!assignees.includes(reviewer)) {
+      botComment += `@${reviewer} `;
+      return true;
+    }
+  });
+  if (unAssignedReviewers.length > 0) {
+    await context.github.issues.addAssigneesToIssue(
+      { ...prParams, assignees: unAssignedReviewers });
+
+    // Ping newly assigned reviewers.
+    await context.github.issues.createComment(
+      { ...prParams, body: botComment });
+  }
+};
+
+module.exports.assignReviewers = async (context) => {
+  // By default, this returns only open PRs
+  const pullRequestsPromiseObj = await context.github.pullRequests.getAll(
+    context.repo({ per_page: 60 })
```

Comment:
Does this mean we will get a paginated response? How is the number 60 chosen?

Reply:
Yes, we will get a paginated response.
I chose 60 because that is what is used in a previous implementation so I decided to follow suit.

Reply:
Which previous implementation are you referring to?

Reply:
https://github.com/oppia/oppiabot/blob/6d723a95549353bb1641325f62efc95c8f420f64/lib/checkMergeConflicts.js#L76

Reply:
Ok!